resources/windows/wix/project.wxs.erb

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="windows-1252"?>
+  <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+    <Product
+      Id="*"
+      UpgradeCode="<%= settings[:upgrade_code] %>"
+      Name="<%= settings[:product_name] %>"
+      Language="1033"
+      Codepage="1252"
+      Version="<%= @version.sub(/\.g[0-9a-z]{7}/, '') %>"
```

Comment:
Oops, this isn't quite safe yet. It should be something closer to `/\.g[0-9a-z]{7}$/` to ensure we're only taking out the last bit. It should be fine, but it's an easy thing to change to prevent possible issues later on